docs/privacy-policy.rst

```diff
@@ -0,0 +1,361 @@
+  `user account <https://readthedocs.org/accounts/edit/>`_.
+
+What information Read the Docs does not collect
+-----------------------------------------------
+
+We do not intentionally collect **sensitive personal information**,
+such as social security numbers, genetic data, health information, or religious information.
+
+Documentation Sites hosted on Read the Docs are public,
+anyone (including us) may view their contents.
+If you have included private or sensitive information in your Documentation Site,
+such as email addresses, that information may be indexed by search engines or used by third parties.
+
+If you're a **child under the age of 13**, you may not have an account on Read the Docs.
+Read the Docs does not knowingly collect information from or direct any of our content specifically to children under 13.
+If we learn or have reason to suspect that you are a user who is under the age of 13, we will unfortunately have to close your account.
+We don't want to discourage you from writing software documentation, but those are the rules.
+
+How we share the information we collect
+---------------------------------------
+
+We **do not** share, sell, rent, or trade User Personal Information with
+third parties for their commercial purposes.
+
+We do not disclose User Personal Information outside Read the Docs,
+except in the situations listed in this section or
+in the section below on compelled disclosure.
+
+We **do** share certain aggregated, non-personally identifying information
+with others about how our users, collectively, use Read the Docs.
+For example, we may compile statistics on the prevalence of
+different types of documentation across Read the Docs.
+
+We **do** host advertising on Documentation Sites.
+This advertising is first-party advertising hosted by Read the Docs.
+We **do not** run any code from advertisers and all ad images are hosted
+on Read the Docs' servers. For more details, see our document on
+:doc:`advertising-details`.
+
+We may share User Personal Information with your permission,
+so we can perform services you have requested.
+
+We may share User Personal Information with a limited number of third-party vendors
+who process it on our behalf to provide or improve our service,
+and who have agreed to privacy restrictions similar to our own Privacy Statement.
+
+
+Third-party vendors
+~~~~~~~~~~~~~~~~~~~
+
+As we mentioned, we may share some information with third-party vendors
+or it may be collected by them on our behalf.
+The information collected and stored by third parties
+is subject to their policies and practices.
+This list will be updated from time to time
+and we encourage you to check back periodically.
+
+Payment processing
+++++++++++++++++++
+
+Should you choose to donate to Read the Docs or purchase a `Gold subscription`_,
+your payment information and details will be processed by Stripe.
+Read the Docs does not store your payment information.
```

Comment:
Also readthedocs.com subscriptions.

Reply:
This privacy policy does not apply to readthedocs.com!